### PR TITLE
Remove unused local.yaml

### PR DESCRIPTION
**Pull request by erkanerol**
Removes creation of local.yaml file, which is unused after #1029

```release-note
NONE
```

### CODE REVIEW

File: hack/make_local.py [hunk 1]
```diff
@@ -13,25 +13,6 @@ def get_env(line):
     return f'{env}={environ.get(env)}'
 
 
-def gen_local_deployments(outdir):
```
Comment (nunnatsa) [inline hack/make_local.py:16]: Please remove the `import yaml` line

**Review by nunnatsa — COMMENTED** [review]
Just a thought: should we rename `hack/make_local.py`, as it's now only generating env vars? Not sure about it, though.

File: hack/make_local.sh [hunk 2]
```diff
@@ -23,7 +23,6 @@ chmod +x _local/deploy.sh
 
 kubectl config set-context --current --namespace=${hco_namespace}
 _local/deploy.sh
-# kubectl apply -f _local/local.yaml
```
[the remark this answers is not in the document]
Comment (erkanerol) [reply hack/make_local.sh:26]: It was commented with previous PR. The comment is being deleted with this PR.

File: hack/make_local.py (hunk 1 above)
Comment (erkanerol) [reply hack/make_local.py:16]: Since I was not using linter for Python, I didn't catch it. Now, I installed a plugin. Thanks for this catch. Fixed.

**Comment by erkanerol** [comment]
> Just a thought: should we rename `hack/make_local.py`, as it's now only generating env vars? Not sure about it, though.

I just renamed as "generate_local_env.py".

**Review by nunnatsa — APPROVED** [review]
/approve

**Pull request merged**